public/docs/_examples/server-communication/dart/pubspec.yaml

```diff
@@ -11,7 +11,7 @@ dependencies:
   http: ^0.11.3+3
   jsonpadding: ^0.1.0
   stream_transformers: ^0.3.0+3
-  http_in_memory_web_api: ^0.0.1
+  http_in_memory_web_api: ^0.2.0
```

Comment:
what is `http_in_memory_web_api`? I guess this is all Dart so I shouldn't care ;-)